pkg/registry/controller/rest_test.go

```diff
@@ -378,7 +379,7 @@ func TestFillCurrentState(t *testing.T) {
 	if controller.Status.Replicas != 2 {
 		t.Errorf("expected 2, got: %d", controller.Status.Replicas)
 	}
-	if !api.Semantic.DeepEqual(fakeLister.s, labels.Set(controller.Spec.Selector).AsSelector()) {
+	if !reflect.DeepEqual(fakeLister.s, labels.Set(controller.Spec.Selector).AsSelector()) {
```

Comment:
FYI - if we get some concept of "internal contstructors" then maybe we can get rid of this and have versioned APIs just use "string" or "IntOrString" - dumb structs.